source/code/core/utils/public/ice/log.hxx

```diff
@@ -2,8 +2,7 @@
 #include <ice/string.hxx>
 #include <ice/log_severity.hxx>
 #include <ice/log_tag.hxx>
-
-#include <fmt/format.h>
+#include <ice/log_formatters.hxx>
```

Comment:
A smaller revisit of the logging API is needed so we can finally support utf8 values. (post-#95)

Reply:
An issues already exists #103